Generals/Code/GameEngine/Source/GameNetwork/Network.cpp

```diff
@@ -665,7 +665,11 @@ void Network::processDestroyPlayerCommand(NetDestroyPlayerCommandMsg *msg)
 	if (pPlayer)
 	{
 		GameMessage *msg = newInstance(GameMessage)(GameMessage::MSG_SELF_DESTRUCT);
+#if RETAIL_COMPATIBLE_CRC
 		msg->appendBooleanArgument(FALSE);
+#else
+		msg->appendBooleanArgument(TRUE);
```

Comment:
Is hard to understand this thanks to the bad message design.

Better write like so:

```cpp
#if RETAIL_COMPATIBLE_CRC
const Bool transferAssets = FALSE;
#else
const Bool transferAssets = TRUE;
#endif
msg->appendBooleanArgument(transferAssets);
```